src/entities/RfqmTransactionSubmissionEntity.ts

```diff
@@ -8,6 +8,7 @@ export enum RfqmTranasctionSubmissionStatus {
     Successful = 'successful',
     Reverted = 'reverted',
     DroppedAndReplaced = 'droppedAndReplaced',
+    Confirmed = 'confirmed',
```

Comment:
confirmed refers to number of blocks deep, so it can be confirmed successful or confirmed reverted I think

Reply:
this will also have implications for the status function, I would update that in this PR as well, see https://github.com/0xProject/0x-api/blob/0f53d64779bfef344c2f7f66fb5bd69006e50c5f/src/services/rfqm_service.ts#L480